### PR TITLE
mapper/api: increase the gunicorn timeout

### PR DESCRIPTION
**Pull request by garbas**
hopefully this fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1483298

### CODE REVIEW

File: nix/lib/default.nix [hunk 1]
```diff
@@ -139,8 +140,8 @@ in rec {
         source = "https://github.com/mozilla/release-services";
         commit = githubCommit;
         build =
-          if taskGroupId != ""
-            then "https://tools.taskcluster.net/groups/${taskGroupId}"
+          if taskGroupId != "" || taskId != ""
```
Comment (La0) [inline nix/lib/default.nix]: That seems unrelated to the timeout

Comment (garbas) [reply nix/lib/default.nix]: yes, unrelated, probable i should call this PR "minor cloudops issues".

Comment (garbas) [reply nix/lib/default.nix]: and this should probably be `&&` instead of `||`

Comment (garbas) [reply nix/lib/default.nix]: we need this since cloudops added a check recently which checks in logs of a task to see if there is a correct docker image digest. without this automatic deployment to production is not possible.
